### ISSUE TITLE
Implement REST API

### ISSUE THREAD

**Issue by swargaraj** [watchtower/watchtower#6]
Develop RESTful API endpoints for frontend interactions such as user authentication, service configuration, and status retrieval.

**Comment by james-tindal** [comment]
I don't think we need an API. React does that all for us these days.

**Comment by swargaraj** [comment]
I agree but for rest api will help to extending to more scopes for like Third-Party Integrations.

**Comment by swargaraj** [comment]
I agree but rest api will help to extending to more scopes for like Third-Party Integrations.
